adafruit_platformdetect/board.py

```diff
@@ -418,6 +418,8 @@ def _imx8mx_id(self) -> Optional[str]:
             return boards.MAAXBOARD
         if "Phanbell" in board_value:
             return boards.CORAL_EDGE_TPU_DEV
+        if "NXP i.MX8MPlus SOM " in board_value:
```

Comment:
Is the trailing space in the name required or something you accidentally added?

Reply:
Purely typo. Thanks for catching it